java-pubsub-authenticated-push/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,9 @@
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <threadsafe>true</threadsafe>
+  <runtime>java8</runtime>
+
+  <env-variables>
+    <env-var name="PUBSUB_TOPIC" value="my-topic-id" />
+    <env-var name="PUBSUB_VERIFICATION_TOKEN" value="1234abc" />
+  </env-variables>
+</appengine-web-app>
```

Comment:
add a new line

Reply:
same way used in examples so not changed